src/schema/v2/artwork/collectorSignals.ts

```diff
@@ -189,6 +190,21 @@ export const CollectorSignals: GraphQLFieldConfig<any, ResolverContext> = {
           return !!artwork.increased_interest_signal
         },
       },
+      runningShowOrFair: {
+        type: Show.type,
+        description:
+          "Most recent running Show or Fair the artwork is currently in, sorted by relevance",
+        resolve: async (artwork, {}, ctx) => {
+          const showOrFair = await await ctx.relatedShowsLoader({
+            artwork: [artwork._id],
+            first: 1,
```

Comment:
I think this data loader makes an API request to Gravity, which I don't _think_ respects `first: 1` as a pagination parameter - did you mean `size: 1`?